lib/bots/syncbot.ts

```diff
@@ -31,6 +33,15 @@ import { Logger, LogLevel } from '../utils/logger';
  * A bot that mirrors threads across services.
  */
 export class SyncBot extends ProcBot {
+	public static getConfig(key: string): string {
+		if (process.env[`SYNCBOT_${key}`]) {
+			return process.env[`SYNCBOT_${key}`];
+		}
+		const fileContents = fs.readFileSync(`./configs/${process.env.CONFIG_TO_LOAD}.yml`, 'utf8');
+		const varsFromFile = yaml.safeLoad(fileContents);
+		return varsFromFile[`SYNCBOT_${key}`];
```

Comment:
So this means that if env vars aren't set you fallback to reading values from the config file? That seems a lot like you could introduce faults into your deployment very easily, by forgetting to set an env var. Surely you should be throwing an error here if the configuration is missing?